src/builder/commands/copy.rs

```diff
@@ -104,6 +111,15 @@ impl BuildStep for Copy {
             // inside the container which is ugly
         }
         hash.field("path", self.path.to_str().unwrap());
+        if let Some(uid) = self.owner_uid {
+            hash.field("owner_uid", uid.to_string());
+        }
+        if let Some(gid) = self.owner_gid {
+            hash.field("owner_gid", gid.to_string());
+        }
+        if let Some(umask) = self.umask {
+            hash.field("umask", umask.to_string());
```

Comment:
A thought experiment: Where will you put a notice that non-specifing umask makes `image-cache-url` partially useless?

I mean people need always specify umask, or otherwise they will complain that `image-cache-url` doesn't work, and most of the time it would be hard to diagnose the problem.